### ISSUE TITLE
When is a new version expected?

### ISSUE THREAD

**Issue by torkale** [jaxxstorm/redphone#7]
I see that some of the latest changes are not part of 0.0.6

**Comment by ghost** [comment]
I agree with this. Made a few issues for myself when using the Sensu statuspage.rb file.

**Comment by andresgutgon** [comment]
Hi I would like also to know what's the status of this gem? Thanks!
